server/src/main/java/io/crate/expression/reference/sys/check/node/DiskWatermarkNodesSysCheck.java

```diff
@@ -46,19 +52,39 @@ abstract class DiskWatermarkNodesSysCheck extends AbstractSysNodeCheck {
 
     @Override
     public boolean isValid() {
-        if (!diskThresholdSettings.isEnabled()) {
-            return true;
-        }
-        DiskUsage leastDiskEstimate = fsService.stats().getLeastDiskEstimate();
```

Comment:
Reverting this usage as this stats claimed to be stale on non-master nodes in https://github.com/elastic/elasticsearch/commit/b75207a09f17c203ca7381f7db4f319225a98070

@mfussenegger could you please take a look as well?